sensor_msgs/Image image
BoundingBox[] boundingBoxes
---
Coloration[] colors